### PR TITLE
fluxctl: init at 1.13.1

### PR DESCRIPTION
**Pull request by Gonzih**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/cluster/fluxctl/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+with import <nixpkgs> {};
```
Comment (yurrriq) [inline pkgs/applications/networking/cluster/fluxctl/default.nix]:
```suggestion
{ stdenv, fetchFromGitHub, buildGoModule }:
```

**Review by yurrriq — CHANGES_REQUESTED** [review]

**Review by yurrriq — APPROVED** [review]

File: pkgs/applications/networking/cluster/fluxctl/default.nix (hunk 1 above)
Comment (Gonzih) [reply pkgs/applications/networking/cluster/fluxctl/default.nix]: fixed

File: pkgs/applications/networking/cluster/fluxctl/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
```
Comment (yurrriq) [inline pkgs/applications/networking/cluster/fluxctl/default.nix]:
```suggestion
{ stdenv, buildGoModule, fetchFromGitHub }:
```

**Review by yurrriq — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/cluster/fluxctl/default.nix (hunk 1 above)
Comment (yurrriq) [reply pkgs/applications/networking/cluster/fluxctl/default.nix]: Same thing here, `s/Package/Module/`.

**Review by yurrriq — APPROVED** [review]

File: pkgs/applications/networking/cluster/fluxctl/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "kubernetes-fluxctl-${version}";
```
Comment (Gonzih) [inline pkgs/applications/networking/cluster/fluxctl/default.nix]:
```suggestion
  pname = "kubernetes-fluxctl";
```

File: pkgs/applications/networking/cluster/fluxctl/default.nix [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  modSha256 = "1q5g9hd0ansdc2acpysf6wi74q50w0psrpyhk4y6mm6kjvhlcn87";
+
+  subPackages = [ "cmd/fluxctl" ];
+
+  meta = with stdenv.lib; {
+    description = "CLI client for Flux, the GitOps Kubernetes operator";
+    homepage = https://github.com/weaveworks/flux;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ Gonzih ];
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (Gonzih) [inline pkgs/applications/networking/cluster/fluxctl/default.nix]:
```suggestion
```

**Comment by yurrriq** [comment]
Should we just call this `fluxctl`? I think the reason `kubernetes-helm` is called that is that there exists another package called `helm`.

**Comment by Gonzih** [comment]
renamed, rebased & squashed

**Review by yurrriq — APPROVED** [review]
LGTM

**Comment by kalbasit** [comment]
@GrahamcOfBorg build fluxctl

**Review by kalbasit — APPROVED** [review]

**Pull request merged**